lib/private/Server.php

```diff
@@ -968,6 +969,7 @@ public function __construct($webRoot, \OC\Config $config) {
 		$this->registerDeprecatedAlias('AsyncCommandBus', IBus::class);
 		/** @deprecated 20.0.0 */
 		$this->registerDeprecatedAlias('TrustedDomainHelper', TrustedDomainHelper::class);
+		$this->registerDeprecatedAlias(ITrustedDomainHelper::class, TrustedDomainHelper::class);
```

Comment:
```suggestion
		$this->registerAlias(ITrustedDomainHelper::class, TrustedDomainHelper::class);
```

This one should not be deprecated I think

Reply:
Opsy